src/common_utils/sf_connector_utils.py

```diff
@@ -35,4 +36,28 @@ def migrate_data(self, params):
         )
         if response.status_code != HTTPStatus.OK:
             raise HTTPError(response.text)
-        return response.json()
+        try:
+            return response.json()
+        except JSONDecodeError:
+            return response.text
+
+    def extract_ddl(self, params):
+        """
+        Initiates snowflake to bigquery ddl extraction and translation
+
+        Args:
+            params (dict): Request payload for migrate-data API
```

Comment:
This can be replaced with extract ddl/translation to be in-line with the functionality

Reply:
Done: https://github.com/GoogleCloudPlatform/data-migration-tool/pull/31/commits/1fd43ecdea21d30d98227716972fa223ec47fb58